src/blueberry-muffins.md

```diff
@@ -0,0 +1,54 @@
+# Blueberry Muffins
+
+![blueberry-muffins](/data/pix/blueberry-muffins.webp)
+
+- ⏲️ Prep time: 20 min
+- 🍳 Baking time: 20 min
+- 🍽️ Servings: 12
+
+## Equipment
+
+-At least 12 muffin tins / A cake tin and optionally a hand mixer
+
+## Ingredients
+
+- 250g / 9 oz Blueberries 
+- 125g / 4½ oz Butter*
+- 150g / 5½ oz Sugar (Of course  you can lower the amount to your liking)
+- 1 Vanilla Pod or Teaspoon of vanilla sugar
+- 2 Eggs
+- 125ml / 4.22 fluid ounces of Milk
+- 300g /  11 oz Flour
+- 2 Teaspoons of Backing Powder
+- 125g / 4½ oz Powdered sugar (optional)
+- Half a lemon (optional)
+*If you don't have Butter, you can substitute it with 100g of Oil / 3½ oz of Oil suited for baking. Notice the measurement in grams / ounces and not milliliter and fluid ounces
+
+## Directions
+
+1. If using a vanilla pod, wash it with water, extract it's vanilla and add it to the molten butter. Put the now empty vanilla pod into the milk. This way every single bit of the vanilla get's extracted
+2. Wash the blueberries
+2. For the liquid ingredients, melt the butter, do not cook it
+3. Add the extracted vanilla / vanilla sugar to the molten butter
+4. Add the milk to the molten butter. Take the vanilla pod out of the milk if you have used one before adding it
+5. Add the two eggs to the molten butter, no need to separate yolk from the egg white
+6. Add the sugar to the molten butter
+7. Mix the aforementioned ingredients up
+8. For the dry ingredients, add the backing powder and salt to the flour
+9. Mix both the liquid ingredients with the dry ingredients, the dough should be thick enough to not be completely fluid but also be just fluid enough to be easily distributed across the muffin tins
+10. The blueberries should have lost most of the water from washing by now, carefully mix them into the dough
+11. Distribute the dough across the muffin tins and put them into the preheated oven (190 °C / 375 Fahrenheit), the muffins should be ready after 20 minutes
+12. Extract the juice of the lemon and slowly add it to the powdered sugar. Mix it with a whisk until you get a thick mass (optional)
+13. Distribute the coating across the top of the muffins and let them cool (optional)
+
+## Cool Tip:
+Muffins are great for sharing at gatherings, but you don't necessary have to waste time and resources on muffin tins. You can always just pour the dough into a cake tin! However, keep in mind that the baking temperature and time should be adjusted. For me 150 °C / 300 Fahrenheit at 50mins. worked the best. 
+
+![blueberry-muffins](/data/pix/blueberry-muffins-01.webp)
+
+## Contribution
+
+- Alexander Platonov - [website](https://alexplatonov.com), [donate](https://alexplatonov/donate.com)
+- Pictures taken by Alexander Platonov
+
+;tags: baking baked sweet cake dessert
```

Comment:
```suggestion
;tags: baking sweet cake dessert
```

I don't really know why we having a `baking` and `baked` tag, but the `baked` tag doesn't actually have any 'baking' on it, so may as well remove it from this recipe.

Reply:
Yea you are right, i've read in the initial readme that i should use already existing ones and thought that "baked" was one of those. But it's probably better to have lesser tags, i'll keep that in mind!